modules/repository/generate.go

```diff
@@ -353,3 +362,12 @@ func GenerateRepository(ctx context.Context, doer, owner *user_model.User, templ
 
 	return generateRepo, nil
 }
+
+// Escapes user input to valid OS filenames
+//
+//	https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file
+func fileNameEscape(s string) string {
+	re := regexp.MustCompile("[/<>:\"\\|?*\n]")
+
+	return re.ReplaceAllString(s, "*")
+}
```

Comment:
Use the regexes from https://github.com/sindresorhus/filename-reserved-regex/blob/main/index.js instead, both of them in one I suppose.

Reply:
Should be all set now 👍

Reply:
Thanks!